SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security Notice
+
+## What?
+
+This is the security notice for all GDS (Government Digital Service) `alphagov` repositories. The notice explains how vulnerabilities should be reported to GDS. At GDS there are cyber security and information assurance teams, as well as security-conscious people within the programmes, that assess and triage all reported vulnerabilities.
```

Comment:
Don't we have other GitHub organisations besides `alphagov`? i.e. https://github.com/openregister

Reply:
Good point. And https://github.com/gds-operations. @deanwilson you recently went and found all these. Are there any others?

Reply:
There's also https://github.com/openregister or similliar which GDS owns.

Reply:
Thanks all - removed `alphagov`.